nixos/modules/services/networking/icecream/daemon.nix

```diff
@@ -0,0 +1,165 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.icecream.daemon;
+  user = "icecc";
+  group = "icecc";
+in {
+
+  ###### interface
+
+  options = {
+
+    services.icecream.daemon = {
+     enable = mkOption {
```

Comment:
Entirely optional, feel free to ignore, but I'm always partial to `mkEnableOption` .